Manifests/ManagedPreferencesApplications/com.secondsonconsulting.renew.plist

```diff
@@ -0,0 +1,543 @@
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		
+		<!-- PayloadUUID -->
+		<!-- This is the PayloadUUID and it will be generated automatically. You should NOT edit this. -->
+		<dict>
+			<key>pfm_description</key>
+			<string>Unique identifier for the payload (format 01234567-89AB-CDEF-0123-456789ABCDEF)</string>
+			<key>pfm_description_reference</key>
+			<string>A globally unique identifier for the payload. The actual content is unimportant, but it must be globally unique. In macOS, you can use uuidgen to generate reasonable UUIDs.</string>
+			<key>pfm_format</key>
+			<string>^[0-9A-Za-z]{8}-[0-9A-Za-z]{4}-[0-9A-Za-z]{4}-[0-9A-Za-z]{4}-[0-9A-Za-z]{12}$</string>
+			<key>pfm_name</key>
+			<string>PayloadUUID</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload UUID</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		
+		<!-- PayloadVersion -->
+		<!-- This is the version of the payload. You should NOT edit this. -->
+		<dict>
+			<key>pfm_default</key>
+			<integer>1</integer>
+			<key>pfm_description</key>
+			<string>The version of the whole configuration profile.</string>
+			<key>pfm_description_reference</key>
+			<string>The version number of the individual payload.
+A profile can consist of payloads with different version numbers. For example, changes to the VPN software in iOS might introduce a new payload version to support additional features, but Mail payload versions would not necessarily change in the same release.</string>
+			<key>pfm_name</key>
+			<string>PayloadVersion</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload Version</string>
+			<key>pfm_type</key>
+			<string>integer</string>
+		</dict>
+		
+		<!-- PayloadOrganization -->
+		<!-- This is the organization of the payload. You should NOT edit this. -->
+		<dict>
+			<key>pfm_description</key>
+			<string>This value describes the issuing organization of the profile, as displayed to the user</string>
+			<key>pfm_name</key>
+			<string>PayloadOrganization</string>
+			<key>pfm_title</key>
+			<string>Payload Organization</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		
+		<!-- *************************** APPLICATION KEYS *************************** -->
+		<!-- You should add all keys this payload can manage below this comment. -->
+
+		<!-- Example: Dictionary -->
+		<!-- This example shows how to define a dictionary -->
+		<dict>
+			<key>pfm_name</key>
+			<string>RequiredArguments</string>
+			<key>pfm_type</key>
+			<string>dictionary</string>
+			<key>pfm_title</key>
+			<string>Required Arguments</string>
+			<key>pfm_description</key>
+			<string>There are four required arguments to make a valid Renew configuration.</string>
+			<key>pfm_description_reference</key>
+			<string>https://github.com/SecondSonConsulting/Renew/wiki/RequiredArguments</string>
```

Comment:
`pfm_description_reference` is generally used for documentation text. I you'd like to use links to your documentation, please use the tag `pfm_documentation_url` instead.

Reply:
Changed all of these as requested